packages/react/test/specs/behaviors/testDefinitions.ts

```diff
@@ -258,6 +258,36 @@ definitions.push({
   },
 })
 
+// Adds attribute 'aria-haspopup=true' to 'trigger' slot if 'contextMenu' property is not set.
```

Comment:
Shouldn't it be `is falsy` ? I assume it has the same behavior for `contextMenu: false` right?

Reply:
I added test for the "false" value below. :)
 
The description I would rather let "if 'contextMenu' property is not set."
Because high probably I expect the usage will be not setting 'contextMenu' prop at all.

Or use as you suggesting:
"if 'contextMenu' property is falsy." 
- it would make sense, but for me is more natural to inform user(consumer) of 'stardust' that "aria-haspopup=true" will be set, when he will not set the "contextMenu" prop...